censusdis/data.py

```diff
@@ -1244,3 +1244,124 @@ def geographies(dataset: str, vintage: VintageType) -> List[List[str]]:
 
 
 variables = VariableCache()
+
+
+def _identify_counties(geo_df: gpd.GeoDataFrame, year: int):
+    """
+    Takes a geodataframe and identifies which US counties the supplied geography overlaps.
+
+    Parameters
+    ----------
+    geo_df
+        A GeoDataFrame containing polygons within the United States
+    year
+        The year for which to fetch geometries. We need this
+        because they change over time.
+
+    Returns
+    -------
+        A list of five digit county FIPS codes.
+    """
+
+    # Some dataframes will contain the county column already
+    if "STATE" in geo_df and "COUNTY" in geo_df:
+        geo_df["FIPS"] = geo_df["STATE"] + geo_df["COUNTY"]
```

Comment:
Instead of modifying the argument dataframe, just store this series in a local variable, e.g. `fips_codes = geo_df["STATE"] + geo_df["COUNTY"]`.